backend/globaleaks/models/serializers.py

```diff
@@ -385,3 +388,68 @@ def serialize_tenant(session, tenant, config=None):
         ret.update(ConfigFactory(session, tenant.id).serialize('tenant'))
 
     return ret
+
+
+def serialize_auditlog(log):
+    return {
+        'date': log.date,
+        'type': log.type,
+        'user_id': log.user_id,
+        'object_id': log.object_id,
+        'data': log.data
+    }
```

Comment:
I'm not sure the type of `log`, but if it is `auditlogResolverModel` in `client/app/src/models/resolvers/auditlog-resolver-model.ts`, `user_id` and `object_id` are optional and might be `undefined`. This works well, but could introduce a potential bug downstream.